cmd/state-installer/cmd.go

```diff
@@ -123,6 +123,19 @@ func main() {
 	logging.Debug("Original Args: %v", os.Args)
 	logging.Debug("Processed Args: %v", processedArgs)
 
+	// Store sessionToken to config
+	for _, envVar := range []string{constants.OverrideSessionTokenEnvVarName, constants.SessionTokenEnvVarName} {
+		sessionToken, ok := os.LookupEnv(envVar)
+		if !ok {
+			continue
+		}
+		err := cfg.Set(anaConst.CfgSessionToken, sessionToken)
+		if err != nil {
+			logging.Error("Unable to set session token: " + errs.JoinMessage(err))
```

Comment:
We _may_ want to report this to rollbar so we can more easily diagnose if the issue of a missing session token comes up again.

Reply:
Yes, that's a good idea.